packages/core/src/extensions/BackgroundColor/BackgroundColorExtension.ts

```diff
@@ -1,32 +1,25 @@
 import { Extension } from "@tiptap/core";
-import { defaultProps } from "../../blocks/defaultProps.js";
+
+import { getAttributeFromDefaultProps } from "../../blocks/defaultProps.js";
 
 export const BackgroundColorExtension = Extension.create({
   name: "blockBackgroundColor",
 
   addGlobalAttributes() {
     return [
       {
-        types: ["blockContainer", "tableCell", "tableHeader"],
```

Comment:
I suppose not, but does this break anything for colors / alignment on custom blocks (as you're changing this from blockcontainer to explicitly naming the block types)

Reply:
Still works as the API (specifically `propsToAttributes`) converts the props from the `propSchema` to TipTap attributes. This extension is just overrides those attributes so we can add unique parse rules.